app/delivery/send_to_providers.py

```diff
@@ -69,6 +69,11 @@ def send_sms_to_provider(notification):
             show_prefix=service.prefix_sms,
         )
 
+        template_content = str(template)
+        if not template_content or template_content.isspace():
```

Comment:
Sorry for hassling again.. can we have this in a utility function (e.g. `is_blank`) and module? 😅